codestyle-config/pom.xml

```diff
@@ -22,16 +22,18 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
   <parent>
-    <groupId>org.apache.apex</groupId>
```

Comment:
Please remove the parent pom changes. They are not related to this change and can be taken up separately (while also changing Apex parent pom).